indigo_api/tests/fixtures.py

```diff
@@ -85,25 +85,27 @@
     <attachments>
       <attachment eId="att_1">
         <doc name="schedule">
+          <heading>Schedule 1</heading>
+          <subheading>Subheading</subheading>
```

Comment:
I believe these should be above (and siblings of) `doc`? see e.g. https://edit.laws.africa/documents/2460/